Sources/String++.swift

```diff
@@ -79,3 +77,30 @@ extension String {
 
 }
 
+
+/// Needed for API compatibility with Scanner in swift-corelibs-foundation
+/// These are exactly the same API as in 
+/// https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/NSScanner.swift
+extension Scanner {
```

Comment:
For the sake of project uniformity, please have the `extension Scanner { ... }` as its own file.